### ISSUE TITLE
Sprint - Oct 3 to Oct 14

### ISSUE THREAD

**Issue by timgl** [PostHog/posthog#12014]
# Global Sprint Planning

## 3 things that might take us down

1.
2.
3.

## Retro: Status of Outcomes from Previous Sprint

1. Text tiles on Dashboards with our team and beta users @pauldambra - hoping for this week

2. re-re-re-naming calculated events @mariusandra - we re-re-re-named, we may re-re-re-re-name. saga continues?
3. Debug and re-run persons on events backfill @yakkomajuri: new backfill has been successful for 2 self-hosted users and the test team on Cloud
4. Deprecate $create_alias @tiina303: ready to communicate deprecation to users, 
5. 🇪🇺 EU plan and build out testing framework for load for new infrastructure (both ingestion and query side) @ellie - did some experiments but no concrete plans. It was optimistic, lots of people off. 
6. Continue improving visibility and making loki logging more scalable @hazzadous - Loki's pretty good, setting other things up
7. Continue supporting person on events rollout where possible @EDsCODE - rolled out to teams that shouldn't have been impacted. Waiting for results of new backfill before rolling out further.
8. List of clean up tasks for team west @liyiy @neilkakkar @EDsCODE - didn't tackle the list, but we did stuff on client library and timestamp issues. 
9. Getting new PlaylistView out for session recordings @alexkim205 - bunch of stuff done, new events filtering, console log, only blocker is UX stuff and fuzzy match. We can get this done this week, we'll roll out in increments.
10. PoC of Billing service @kappa90 - super deep on this. It runs in a flat in rome. PoC is there, now moving to MVP.

## Retro: What can we do better next sprint?

1. More time in SF
2. Karl: it's shifted from deployments. More things around filtering/insights. A lot of bugs, a lot of obvious missing features.
3.
4.
5.

## Support hero this sprint

Week 1: James G
Week 2: Tiina

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1. Curation: 🎨 Make it easy to quickly create an AARRR-inspired dashboard. @pauldambra
Why? Make it easier for people to get started with analytics and surface interesting metrics.
How? Adding metadata (A, A, R, R or R?) to actions, using this knowledge to guide the user through the app and create insights and dashboards via curated templates.
2. Exploration: 🧭 Build v0.1 of the "data explorer" / "universal search". @mariusandra
Why? It's hard to analyse relations between pieces of data.
3. 🇪🇺 cloud needs productionalization + billing @ellie 
why? Paying EU customers have a good experience
4. Digitalocean single click + hobby deploy - make sure that works - testing if neccessary @fuziontech
why? Most(?) users use digital ocean + hobby deploy, and it's a bad experience.
5. MVP for app metrics (how many events delivered, error rates) @macobo (pair: @tiina303)
Why: This is effectively a requirement for CDPs and will give us and users more confidence in the pipeline
6. Ensure robustness of async handlers (jobs and scheduled tasks) @hazzadous (pair: @yakkomajuri)
Why: So that apps can work reliably even when services are undergoing issues
7. Enabling Session Recording billing via BillingService @benjackwhite
why? so we can charge for session recordings + eu cloud
8. Event filtering in Session Recordings & fuzzy filtering of events (MVP) @alexkim205
why? we already have it + only mechanism to filter recordings down
9. Revisit metahog (direct sql access) and set up for MVP deployment on self-hosted @EDsCODE 
why? To help power users get more out of posthog
10. Cleanup and growth tasks for experiments @liyiy
why? boost usage of experiments

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Bugs

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
Team Product Analytics

## Retro

## Hang over items from previous sprint

- Renaming calculated events back to actions: **done**
- Text tiles to dashboards: *still this week*

## Planning

### High priority

OKR: [Nail Data Exploration.](https://github.com/PostHog/posthog.com/pull/4215/files)
There are two sides to this: exploration and curation.

- Curation: 🎨 Make it easy to quickly create an AARRR-inspired dashboard. @pauldambra 
Why? Make it easier for people to get started with analytics and surface interesting metrics.
How? Adding metadata (A, A, R, R or R?) to actions, using this knowledge to guide the user through the app and create insights and dashboards via curated templates.

- Exploration: 🧭 Build v0.1 of the "data explorer" / "universal search". @mariusandra 
Why? It's hard to analyse relations between pieces of data.


### Bugs

- https://github.com/PostHog/posthog/issues/11349
- https://github.com/PostHog/posthog/issues/11727
- https://github.com/PostHog/posthog/issues/9478
- https://github.com/PostHog/posthog/issues/7468
- https://github.com/PostHog/posthog/issues/11887
- https://github.com/PostHog/posthog/issues/8408
- https://github.com/PostHog/posthog/issues/10275

### Low priority / side quests

- 2FA for PostHog @Twixes 
- Inject custom JS into your website via posthog-js @mariusandra

**Comment by fuziontech** [comment]
Team Platform

### Previous sprint

Hang over items from previous sprint
- 🤔 There are still some PG and CH things going on. Not gotten to the bottom of issues, but at least have more visibility into it.
Planning
- 📈 - 🇪🇺 EU plan and build out testing framework for load for new infrastructure (both ingestion and query side)
- ✅ - 🇪🇺 Finalize list of Beta users for EU infrastructure
- ✅ - Continue improving visibility and making loki logging more scalable.
- ✅ - Set Q4 goals

## Retro

- @ellie - It was good. SF is dope <- James's words. Have empathy trying to talk to EU folk
- @fuziontech - PEOPLE WERE IN SF! It was rad. I love having people local to jam with. Talked with @timgl about some really interesting things we can do around CH + Query interface infra.
- @joethreepwood - Walking holiday 70k of hiking. Did 50k in 10 hours just the week before. 

### Planning bits
@guidoiaquinti returns on Monday!
@fuziontech support hero beginning Oct 1 (next week)

## Hang over items from previous sprint
- 📈 - 🇪🇺 EU plan and build out testing framework for load for new infrastructure (both ingestion and query side) @guidoiaquinti 

## Planning

- 🇪🇺 cloud needs productionalization. Beta users will have feedback. We will need to address that. @ellie 
- 🇪🇺 cloud billing needs finalization. Working with Billing service team to be the beta testers there @ellie 
- Daniel 🇨🇴 - onboarding plan @ellie (starts next sprint technically, but need a plan)
- Digitalocean single click - make sure that works - testing if neccessary @fuziontech 
- Hobby deploy - complete end to end testing on Digitalocean on a fresh droplet @fuziontech 
- Low key offboard @hazzadous 🙈

**Comment by yakkomajuri** [comment]
# Team Ingestion

## Planning

**Note:** we will also trigger the backfill for all Cloud teams

- MVP for app metrics @macobo (pair: @tiina303)
**Why:** This is effectively a requirement for CDPs and will give us and users more confidence in the pipeline

- Ensure robustness of async handlers (jobs and scheduled tasks) @hazzadous (pair: @yakkomajuri)
    1. Scheduled tasks don’t crash async server
    2. Jobs aren’t lost
   
**Why:** So that apps can work reliably even when services are undergoing issues

Side quest: @lharries & @yakkomajuri to work on CDP roadmap & schedule initial customer interviews

## Hang over items

- Communicate create_alias functionality change to users @tiina303 

## Retro

- @macobo 
    - Support hero required a lot of work earlier in the week, less towards the end
    - Excited to have Luke on board

- @yakkomajuri 
    - Appreciate the context-setting from Luke
    - Happy with the OKRs planning: efficient meetings
    - Happy that the new backfill works!
        - We should make postchecks standard practice
    - Glad we found the issue with the async server pods restarting
        - How could we have caught this earlier?
            - Set up alert --> what liveness metrics should we track? @yakkomajuri 
            - We should log every important operation!
            - Further splitting the plugin server into more services _could_ help us catch, debug, and manage the "internal services" (jobs, schedule, etc.) --> what are the tradeoffs?
   
- @tiina303 
    - Happy to be back
    - OKRs planning was good

- @hazzadous 
    - Happy we've unified the Docker images
    - Satisfied with the sprint
    - Disappointed with Loki
    - Happy to come over to ingestion
    - EU Cloud 🎉 

- General point: Yakko could have handled work/sickness better

**Comment by benjackwhite** [comment]
# Team Rickordings 

## Retro



- 

## Hang over items from previous sprint



- Rick did an amazing job at putting his brain in ours 🙌 
- Final bits for the playlist view
- (waiting) - events_summary usage rollout
  - Run some test queries on events_summary to ensure the data looks good and is queryable

## Planning



### High priority
- Enabling Session Recording billing via BillingService @benjackwhite 
- Event filtering in Session Recordings & fuzzy filtering of events (MVP) @alexkim205 

### Bugs

-

### Low priority / side quests
- Encouragement UI for Session Recordings @alexkim205

**Comment by EDsCODE** [comment]
# Team West 

## Retro



- person on events - still waiting on new backfill
- clean up bugs - timestamp and date operator work

## Hang over items from previous sprint



- person on events

## Planning



### High priority
- Revisit metahog and set up for MVP deployment on self-hosted @EDsCODE 
     - Develop plan for what deployment on cloud looks like
- Cleanup and growth tasks for experiments @liyiy 
     - Make it easier to go from feature flag to experiments
     - Update code snippets
     - https://github.com/PostHog/posthog/issues/10426
     - *Revisit on friday once OKR is ironed out
- Customer calls + competitor analysis (@EDsCODE  + @liyiy  + @lharries )
### Bugs

-
